src/components/Layouts.jsx/MainLayout.jsx

```
@@ -702,48 +740,194 @@ const MainLayout = (props) => {
+                                                ? "Pause"
+                                                : "Play"
+                                            }
+                                            style={{ height: "30px" }}
+                                          />
+                                        </button>
+                                      ) : (
+                                        <Box></Box>
+                                      )}
+                                      <audio
+                                        ref={(el) =>
+                                          (audioRefs.current[index] = el)
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)